### ISSUE TITLE
How to use tables extension?

### ISSUE THREAD

**Issue by zoujiaqing** [huntlabs/hunt-markdown#1]
@gaoxincheng

**Comment by gaoxincheng** [comment]
Refer to the sample [directory](https://github.com/huntlabs/hunt-markdown/tree/master/example/resources) ,which involves converting common HTML tags.

**Comment by zoujiaqing** [comment]
@gaoxincheng 

Like this code?
```D
import hunt.markdown.node.Node;
import hunt.markdown.parser.Parser;
import hunt.markdown.renderer.html.HtmlRenderer;

import hunt.collection.Set;

Parser parser = Parser.builder().extensions(Collections.singleton(TableExtension.create())).build();
Node document = parser.parse("tables markdown code in here");
HtmlRenderer renderer = HtmlRenderer.builder().build();
renderer.render(document);
```

**Comment by gaoxincheng** [comment]
Emm... You can try that.

**Comment by zoujiaqing** [comment]
Thanks!

**Comment by zoujiaqing** [comment]
Table example is work:
```D
import std.stdio;

import hunt.markdown.Extension;
import hunt.markdown.node.Node;
import hunt.markdown.parser.Parser;
import hunt.markdown.renderer.html.HtmlRenderer;
import hunt.markdown.ext.table;

import hunt.collection.Collections;

void main()
{
	string markdown = `
## Additional package dependencies
| package | version | purpose |
|--------|--------|--------|
| hunt-trace |  0.2.0 above |  Tracing for API requests  |
| hunt-security |  0.2.0 above |  Some core APIs for security  |

`;


	auto extensions = Collections.singleton(TableExtension.create());

	Parser parser = Parser.builder().extensions(extensions).build();
	Node document = parser.parse(markdown);
	HtmlRenderer renderer = HtmlRenderer.builder().extensions(extensions).build();

	string content = renderer.render(document);

	writeln(content);
}
```

**Comment by zoujiaqing** [comment]
This bug fixed in version 1.0.2 :)
